src/bokeh/transform.py

```diff
@@ -119,6 +121,46 @@ def dodge(field_name: str, value: float, range: Range | None = None) -> Field:
     '''
     return Field(field_name, Dodge(value=value, range=range))
 
+def eqhist_cmap(field_name: str, palette: Sequence[ColorLike], low: float, high: float,
+        low_color: ColorLike | None = None, high_color: ColorLike | None = None, nan_color: ColorLike = "gray") -> Field:
```

Comment:
very minor style nit, but I'd prefer to see long function signatures with one param per line:
```suggestion
def eqhist_cmap(field_name: str, 
                palette: Sequence[ColorLike], 
                low: float, 
                high: float,
                low_color: ColorLike | None = None, 
                high_color: ColorLike | None = None, 
                nan_color: ColorLike = "gray") -> Field:
```

Reply:
If we are suggesting styling changes, then I would strongly recommend this instead:
```suggestion
def eqhist_cmap(
    field_name: str,
    palette: Sequence[ColorLike],
    low: float,
    high: float,
    low_color: ColorLike | None = None,
    high_color: ColorLike | None = None,
    nan_color: ColorLike = "gray",
) -> Field:
```
because why waste so much space on the left. This would be even more visible with a function with a longer name.

Reply:
I was wondering, if I'm changing the styling of the `eqhist_cmap` function, why not change the styling of log_cmap/linear_cmap as well. I wouldn't mind

Reply:
Fine with either of the above and also fine with changing the other functions to match.